### DISCUSSION TITLE
Stucked at codespace saving file

### DISCUSSION THREAD

**Discussion by VisualLaser10New** [community/community#131299] [Codespaces]
### Select Topic Area

Bug

### Body

Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.92.0-insider
- OS Version: Windows 10 22H2 (19045)

After a while (not precise) vs-code-insiders is connected to a codespace, while saving, remains stucked at "Saving file ..." without any possibilities to save/open anothers file or compile a code. I can continue editing the file, but newest changes won't be saved.

Every time only possibility is to close vs-code, stop codespace and restart it, by losting last work.
I tried to create a new codespace but the problem persist.

Steps to Reproduce:
There are no steps

**Comment by taslim-a-hussain** [comment]
**Hey VisualLaser10New,**

Sorry to hear you're having trouble with saving files in Codespaces. Here are a few things you might try to resolve the issue:

1. **Update VS Code Insiders:** Make sure you're running the latest version of VS Code Insiders. Sometimes, bugs are fixed in newer releases.
2. **Check Network Stability:** Ensure that your network connection is stable, as network issues can sometimes cause problems with Codespaces.
3. **Increase Resources:** Sometimes, Codespaces can run into resource limits. Try increasing the resources allocated to your Codespace in the settings.
4. **Clear Cache:** Clear the VS Code cache and try again. Sometimes, corrupted cache files can cause issues.
5. **Log Out and Back In:** Log out of VS Code Insiders and your Codespace, then log back in to refresh the connection.

Hope this helps! Let us know if you find a solution or need further assistance.

Good luck!

**Reply by VisualLaser10New** [reply]
Hi, I want to say vs code insiders is up to date and I already tried to delete cache of it (i deleted "%appdata%\code insider" folder, if there is another folder I can delete tell me). 
Anyway the connection is stable and enough fast (100Mbps), and by deleting cache i just logged out from vs-code account and relogged in. As you said, I tried to increase the codespace resources but never changes.

**Reply by taslim-a-hussain** [reply]
Thanks for trying out the troubleshooting steps I suggested earlier. To help you further, could you please try the following:

1. Although you mentioned that the issue occurs even when all extensions are disabled, it's possible that a specific extension is causing the problem. How about disabling extensions one by one and testing if the issue persists?

2. I know you already tried creating a new Codespace, but how about creating a new Codespace environment with a different configuration to see if the issue is specific to your current environment?

**Comment by angelusmcnally** [comment]
I run into this issue every few days as well and it's super frustrating -- is there any progress on this?

**Comment by mchubbit** [comment]
Try disabling all your extensions and see if that resolves the problem. If it does, then you can try re-enabling them one by one to see which one might be causing the issue. You can try updating to the latest stable version of VS Code and see if that helps. The Insider build you're using might have some bugs.

**Comment by subwaymenu6** [comment]
It sounds like a frustrating issue with VS Code Insiders where files won’t save, and you lose progress after restarting. Since it happens even with all extensions disabled, it could be a bug or a configuration issue with your codespace or VS Code version. Try checking for updates or reinstalling VS Code. If the issue continues, reporting it with logs on GitHub might help get a fix—kind of like when you're troubleshooting something complex, like managing [subway premium subs](https://subwaymenu.de/) options for a seamless experience.

**Comment by VisualLaser10New** [comment]
The problem has been fixed for me by updating, with latest update
